staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -0,0 +1,336 @@
+				Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
+					AssuredConcurrencyShares: 30,
+					LimitResponse: flowcontrol.LimitResponse{
+						Type: flowcontrol.LimitResponseTypeQueue,
+						Queuing: &flowcontrol.QueuingConfiguration{
+							Queues:           128,
+							HandSize:         6,
+							QueueLengthLimit: 100,
+						},
+					},
+				},
+			}),
+		// catch-all priority-level
+		DefaultPriorityLevelConfigurationDefault,
+		// exempt priority-level
+		DefaultPriorityLevelConfigurationExempt,
+	}
+}
+
+// PredefinedFlowSchemas returns instances of the FlowSchema
+// objects shown in the Example Configuration section of the KEP
+func PredefinedFlowSchemas() []*flowcontrol.FlowSchema {
+	verbAll := []string{flowcontrol.VerbAll}
+	apiGroupAll := []string{flowcontrol.APIGroupAll}
+	resourceAll := []string{flowcontrol.ResourceAll}
+	namespaceEvery := []string{flowcontrol.NamespaceEvery}
+	resourceRuleAll := resourceRule(verbAll, apiGroupAll, resourceAll, namespaceEvery, true)
+	nonResourceRuleAll := nonResourceRule(verbAll, []string{flowcontrol.NonResourceAll})
+	return []*flowcontrol.FlowSchema{
+		newFlowSchema(
+			"system-nodes", "system", 2500,
+			flowcontrol.FlowDistinguisherMethodByUserType,
+			flowcontrol.PolicyRulesWithSubjects{
+				Subjects:         groups(user.NodesGroup),
+				ResourceRules:    []flowcontrol.ResourcePolicyRule{resourceRuleAll},
+				NonResourceRules: []flowcontrol.NonResourcePolicyRule{nonResourceRuleAll},
+			},
+			flowcontrol.PolicyRulesWithSubjects{
+				Subjects: kubeSystemServiceAccount(flowcontrol.NameAll),
+				ResourceRules: []flowcontrol.ResourcePolicyRule{
+					resourceRule(verbAll, []string{corev1.GroupName}, []string{"endpoints", "configmaps"}, namespaceEvery, false),
+					resourceRule(verbAll, []string{coordinationv1.GroupName}, []string{"leases"}, namespaceEvery, false),
+				},
+			},
+		),
+		newFlowSchema("system-leader-election", "system", 2500,
+			flowcontrol.FlowDistinguisherMethodByUserType,
+			flowcontrol.PolicyRulesWithSubjects{
+				Subjects: users(user.KubeControllerManager, user.KubeScheduler),
+				ResourceRules: []flowcontrol.ResourcePolicyRule{
+					resourceRule(verbAll, []string{corev1.GroupName}, []string{"endpoints", "configmaps"}, namespaceEvery, false),
+					resourceRule(verbAll, []string{coordinationv1.GroupName}, []string{"leases"}, namespaceEvery, false),
+				},
+			},
+		),
+		newFlowSchema("workload-leader-election", "system", 2500,
+			flowcontrol.FlowDistinguisherMethodByUserType,
+			flowcontrol.PolicyRulesWithSubjects{
+				Subjects: kubeSystemServiceAccount(flowcontrol.NameAll),
+				ResourceRules: []flowcontrol.ResourcePolicyRule{
+					resourceRule(verbAll, []string{corev1.GroupName}, []string{"endpoints", "configmaps"}, namespaceEvery, false),
+					resourceRule(verbAll, []string{coordinationv1.GroupName}, []string{"leases"}, namespaceEvery, false),
+				},
+			},
+		),
+		newFlowSchema("kube-controller-manager", "workload-high", 3500,
```

Comment:
I'm puzzled about what "workload-high" means; I thought it was for users to use until now.

Reply:
i added some godoc on those pre-defined FS/PL resources